CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Changed
 
+- (react-native): Update bugsnag-cocoa to v6.4.0
```

Comment:
@bengourley looks like a RN release containing bugsnag-cocoa 6.3.0 never went out?!

If so, we should merge the changelog entries, and be sure to also include a summary of the changes in bugsnag-cocoa 6.4.0

Reply:
Yeah we held off on that while waiting for the dust to settle on the release stage validation stuff.

Reply:
@kstenerud it would be good to merge the changelog entries please 🙏

Reply:
Added under this line